subsys/net/l2/ethernet/dsa/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+zephyr_library_include_directories(. ${ZEPHYR_BASE}/subsys/net/ip)
```

Comment:
Why is `ip` a requirement for `dsa` ? Note that Ethernet should have no dependency on higher level stacks, because some day we might want to use it without IP / TCP / UDP also..

Reply:
I think that it was a leftover from some testing. I've already removed it. Thanks for your feedback.